caddytest/integration/caddyfile_adapt/reverse_proxy_health_headers.txt

```diff
@@ -0,0 +1,49 @@
+:8884
+
+reverse_proxy 127.0.0.1:65535 {
+    health_headers {
+        Host example.com
+        X-Header-Keys VbG4NZwWnipo 335Q9/MhqcNU3s2TO
+    }
+}
+----------
+{
+    "apps": {
+        "http": {
+            "servers": {
```

Comment:
Use tabs, not spaces, in this file.

Reply:
yes, I forget to run test case before push.